pkg/homedir/homedir_unix.go

```diff
@@ -94,3 +99,98 @@ func stick(f string) error {
 	m |= os.ModeSticky
 	return os.Chmod(f, m)
 }
+
+var (
+	rootlessConfigHomeDirOnce sync.Once
+	rootlessConfigHomeDir     string
+	rootlessRuntimeDirOnce    sync.Once
+	rootlessRuntimeDir        string
+)
+
+// GetRootlessConfigHomeDir returns the config home directory when running as non root
```

Comment:
Is this supposed to be used as an alternative to `GetConfigHome()`?  When?

Reply:
the documentation of GetConfigHome() is very clear on the expectation of that function, and I didn't want to break it.  I am not sure what is the best way to proceed.

Are you okay if we add the extra feature of trying to guess the ConfigHomeDir to the existing `GetConfigHome()` and not add a new function?